docs/pages/access-controls/reference.mdx

```diff
@@ -12,7 +12,7 @@ controls (RBAC) in your Teleport cluster.
 A Teleport `role` works by having two lists of rules: `allow` rules and `deny` rules.
 When declaring access rules, keep in mind the following:
 
-- Everything is denied by default.
+- Nothing is allowed by default.
```

Comment:
I think this is an important distinction, when a role explicitly denies something that means that no other role can allow access to it, even through an access request.